speechbrain/inference/interfaces.py

```diff
@@ -69,66 +130,43 @@ def foreign_class(
 
     Arguments
     ---------
+    cls : Type[Pretrained]
+        The class to construct an instance of, usually a sub-type of Pretrained
     source : str or Path or FetchSource
         The location to use for finding the model. See
         ``speechbrain.utils.fetching.fetch`` for details.
     hparams_file : str
         The name of the hyperparameters file to use for constructing
         the modules necessary for inference. Must contain two keys:
         "modules" and "pretrainer", as described.
```

Comment:
I guess same here?

Reply:
What did you mean by this exactly? That this argument should be explained better?